packages/typescript/ai-memory/tests/redis.test.ts

```
@@ -0,0 +1,14 @@
+// @ts-expect-error -- ioredis-mock has no bundled types and we don't need them
+//   here; the contract test only exercises the RedisLike subset that
+//   redisMemoryAdapter consumes (cast to `never` below).
+import RedisMock from 'ioredis-mock'
+import { runMemoryAdapterContract } from './contract'
+import { redisMemoryAdapter } from '../src/adapters/redis'
```

ESLint

[error] `../src/adapters/redis` import should occur before import of `./contract`

(import/order)